python_ta/reporters/node_printers.py

```diff
@@ -153,6 +205,328 @@ def render_blank_line(line):
     yield (line + 1, slice(None, None), LineType.ERROR, " " * 28)
 
 
+def render_pep8_errors_e101(msg, _node, source_lines=None):
+    """Render a PEP8 indentation contains mixed spaces and tabs message."""
+    line = msg.line - 1
+    curr_idx = 0
+    while source_lines[line][curr_idx].isspace():
```

Comment:
In general don't use while loops to detect leading indentation. Instead, use `len(s) - len(s.lstrip())`